src/components/Accounts/Menu.js

```diff
@@ -14,7 +14,7 @@ import { getFullName } from '../../util';
 const Menu = (props) => {
   const { user, showFilters, filters, balance, selected, actions, query } = props;
   const outstanding = parseFloat(balance).toFixed(2);
-
+  const showSelected = (selected !== 0 && selected !== parseFloat(0).toFixed(2)) && outstanding > parseFloat(0).toFixed(2);
```

Comment:
this string looks so long
maybe it is better to place it on separate lines, somehow like:
```
const showSelected = (selected !== 0 && selected !== parseFloat(0).toFixed(2)) 
         && outstanding > parseFloat(0).toFixed(2);
```